<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fct_comp_7</class>
 <widget class="QDialog" name="fct_comp_7">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion de representation</string>
  </property>
  <widget class="QTableWidget" name="table_fct_comp_7">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>521</width>
     <height>431</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>128</number>
   </attribute>
   <column>
    <property name="text">
     <string>noSpec</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>dateRep</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>promoRep</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>nbPlacesDispo</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="message_fct_comp_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>10</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="noSpec_fct_combo_7_2_1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7_2_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Selectionner la represenation à supprimer /modifier :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="dateRep_combo_fct_comp_7_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="noSpec_label_7_2_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>59</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>noSpec :</string>
   </property>
  </widget>
  <widget class="QLabel" name="dateRep_label_7_2_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>59</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>dateRep :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_edit_7_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>308</x>
     <y>180</y>
     <width>110</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 85, 0);</string>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="noSpec_fct_combo_7_2_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_supprimer">
   <property name="geometry">
    <rect>
     <x>138</x>
     <y>180</y>
     <width>110</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="dateRep_label_7_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>59</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>dateRep :</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateRepEdit_fct_comp_7_1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="date">
    <date>
     <year>2019</year>
     <month>11</month>
     <day>25</day>
    </date>
   </property>
   <property name="minimumDateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2019</year>
     <month>11</month>
     <day>25</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>dd/MM/yyyy HH:mm</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="noSpec_label_7_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>59</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>noSpec :</string>
   </property>
  </widget>
  <widget class="QLabel" name="promo_label_7_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>promoRep :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ajouter une nouvelle représentation</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="promoRep_lineEdit_fct_comp_7_1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>340</y>
     <width>153</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="cursorPosition">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>138</x>
     <y>380</y>
     <width>110</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 85, 0);</string>
   </property>
   <property name="text">
    <string>Ajouter </string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="noSpec_fct_combo_7_1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="mesage_fct_comp_7_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="promo_label_7_2_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>promoRep :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="promoRep_lineEdit_fct_comp_7_2_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>140</y>
     <width>153</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="promoRep_lineEdit_fct_comp_7_2">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>140</y>
     <width>153</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateRepEdit_fct_comp_7_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2019</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="minimumDateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2019</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_supprimer</sender>
   <signal>clicked()</signal>
   <receiver>fct_comp_7</receiver>
   <slot>deleteRepresentation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_edit_7_2</sender>
   <signal>clicked()</signal>
   <receiver>fct_comp_7</receiver>
   <slot>editRepresentation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_1</sender>
   <signal>clicked()</signal>
   <receiver>fct_comp_7</receiver>
   <slot>addRepresentation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>noSpec_fct_combo_7_2_1</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>fct_comp_7</receiver>
   <slot>refreshDateRep()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateRep_combo_fct_comp_7_2</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>fct_comp_7</receiver>
   <slot>refreshPromoRep()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>open_fct_comp_8()</slot>
  <slot>refreshResult()</slot>
  <slot>refreshResult1()</slot>
  <slot>refreshDateRep()</slot>
  <slot>deleteRep()</slot>
  <slot>refreshcomboSpec1()</slot>
  <slot>addRepresentation()</slot>
  <slot>open_fct_comp_7_1()</slot>
  <slot>open_fct_comp_7_2()</slot>
  <slot>open_fct_comp_7_3()</slot>
  <slot>editRepresentation()</slot>
  <slot>deleteRepresentation()</slot>
  <slot>refreshPromoRep()</slot>
 </slots>
</ui>
